.travis.yml

```diff
@@ -2,31 +2,41 @@ matrix:
   include:
     - env: python2-linux
       os: linux
+      virtualenv:
+        system_site_packages: true
+      language: python
       python: '2.7'
       script:
+        - pip install -r requirements-gui-dev.txt
         - coverage run setup.py test && mv .coverage .coverage.core
         - coverage run gui/build.py test && mv .coverage .coverage.gui
         - coverage combine
         - pip install coveralls; coveralls
         - python gui/build.py bundle
     - env: python3-linux
       os: linux
-      python: '3.5'
+      language: python
+      python: '3.6'
```

Comment:
We should support python 3.5 and up, since that's the last supported version (which we also use extensively in our internal automation systems)